contributors/devel/architectural-roadmap.md

```diff
@@ -478,7 +478,7 @@ routing APIs and functions include:
   as a separate process, on or outside the cluster.
 
 * NIY: A
-  [rescheduler](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/rescheduling.md),
+  [rescheduler](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/scheduling/rescheduling.md),
```

Comment:
same repo. This can be:
`[rescheduler](/contributors/design-proposals/scheduling/rescheduling.md)`